testkit/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/testkit/ServiceTest.scala

```diff
@@ -285,7 +285,7 @@ object ServiceTest {
       } else if (setup.cluster) {
         ClusterConfigMap
       } else {
-        Map.empty
+        BasicConfigMap
```

Comment:
Hum... so, `BasicConfigMap` still has ` "lagom.akka.management.enabled" -> "off"` meaning that now Akka Management will be disabled for tests (which is the opposite of production). Is this to avoid port conflicts?

Reply:
More or less. 

Akka Management will only start by default in prod and the reason is to avoid port conflicts, especially in Dev. 
https://github.com/lagom/lagom/blob/master/akka-management/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/server/AkkaManagementComponents.scala#L23-L28

Later, we should generate a dynamic, but stable, port for akka-management in the same way we do for http and https ports of Lagom services. But this can wait for after 1.5.0.

The reason for adding that extra config in the tests is that we need to read the config even before we decide if we are in Prod or not. But tests based on `ServiceTest` do not read the conf from the classpath and therefore don't get the property from the reference.conf. Those tests use a config that is loaded programmatically. 

The consequence is that those tests fail because `lagom.akka.management.enabled` is not defined. 

We could have set it to `on` and the result would have been the same. I choose to set it to `off` for two reasons: 1) it reflects better, although not necessary, what happens in Dev and Tests: no akka-mngt; 2) when we add dynamic port assignment in Dev, we may still want to disable it in tests. Unless someone wants to test the akka-mngt itself, it has no much value to have each test starting and stopping an http server.